int32 empty

--------------------------------------

sensor_msgs/JointState object_pose